Userland/Libraries/LibWeb/DOM/ElementFactory.cpp

```
@@ -474,6 +476,44 @@ static WebIDL::ExceptionOr<JS::GCPtr<SVG::SVGElement>> create_svg_element(JS::Re
     return nullptr;
 }
 
+static WebIDL::ExceptionOr<JS::GCPtr<MathML::MathMLElement>> create_mathml_element(JS::Realm& realm, Document& document, QualifiedName qualified_name)
+{
+    auto const& local_name = FlyString::from_deprecated_fly_string(qualified_name.local_name()).release_value_but_fixme_should_propagate_errors();
```

You can propagate this error with `TRY_OR_THROW_OOM(realm.vm(), ...)`